packages/types/src/index.ts

```diff
@@ -74,7 +74,7 @@ export interface JellyAdapter {
 
     parseFromNative(amount: string): string | number;
 
-    formatInput(data: UserInputSwap, receiver: string): ContractSwap;
+    formatInput(data: UserInputSwap, receiver: string, blockNumber?: number): ContractSwap;
```

Comment:
Remove `blockNumber` from the `formatInput` interface.

Reply:
This change affects all other blockchain implementation and it's not required.

Reply:
It is an optional param, the old implementations should work